openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -116,6 +149,88 @@ public void configureEndpoint(Routing.Rules rules) {
                 .get(webContext, this::prepareResponse);
     }
 
+    static synchronized SnakeYAMLParserHelper<ExpandedTypeDescription> helper() {
+        if (helper == null) {
+            helper = SnakeYAMLParserHelper.create(ExpandedTypeDescription::create);
+            adjustTypeDescriptions(helper.types());
+        }
+        return helper;
+    }
+
+    static Map<Class<?>, ExpandedTypeDescription> buildImplsToTypes(SnakeYAMLParserHelper<ExpandedTypeDescription> helper) {
+        return helper.entrySet().stream()
```

Comment:
Consider wrapping this in `Collections.unmodifiableMap()`.

Reply:
Done